### DISCUSSION TITLE
Utilisation des activités dans le modèle

### DISCUSSION THREAD

**Discussion by jfcjm** [dbarzin/mercator#125] [Ideas]
Bonjour,

Bien que les activités apparaissent dans la doc ANSSI, je ne vois pas bien leur valeur ajoutée dans la cartographie.

Elle sont en relation avec les processus d'une part et les opérations de l'autre. Comme il n'y a aucune autre information qui leur soit liée (pas d'acteur, pas de responsable, pas d'utilisateur, pas d'application, pas d'attribut de sécurité), il semblerait que ces activités n'existent que pour introduire un niveau d'indirection hiérarchique qui ne semble utile que pour grouper visuellement des opérations

Sachant qu'il y a déjà les niveaux opérations et tâches pour modéliser les processus, pourrait-on envisager de rendre leur utilisation facultative ?

Jean-Marie.

**Comment by VinceBlood** [comment]
Je bloque également sur cette partie, principalement sur Opération.

Un processus est un ensemble d'activés et de taches.
Une activité (sous-processus) est un ensemble de taches réutilisables.

Une activité va se nommer comme un processus "Gestion de ...", "Préparation de ..."
Une tache va se nommer "Creer ...", "Commander ..."

Normalement Opération est un constituant d'une tâche or l'ANSSI le place au dessus de la tâche.

On notera que le Guide de l'ANSSI autorise un lien direct entre Activité et tache (sans passer par Opération) lorsque l'on est à un niveau de maturité 1 et 2.

J'avoue je suis un peu perdu.

**Comment by dbarzin** [comment]
Seriez-vous d'accord de supprimer la notion d'activité du modèle de données ?

**Reply by jfcjm** [reply]
@VinceBlood: 
>On notera que le Guide de l'ANSSI autorise un lien direct entre Activité et tache (sans passer par Opération) lorsque l'on est à un niveau de maturité 1 et 2

Je ne comprends pas votre phrase, les Activités n'apparaissent qu'au niveau 3 dans la doc ANSSI (tableau p 34).

De manière générale : si tout le monde semble s'accorde sur l'existence des processus, la décomposition des dits processus diverge  malheureusement en général rapidement que ce soit sur les noms utilisés (opération/tâches) ou sur la profondeur des niveaux de structuration :)



@dbarzin: IMHO, ce seraient plutôt les activités qui pourraient disparaître au regard du petit nombre d'associations dans lesquelles elles interviennent.

J.-M.

**Reply by VinceBlood** [reply]
En réponse à jfcjm, j'ai peut-être mal rédigé mon message initial.
Page 35 :
Un processus contient des activités (ou des opérations qui les composent, si les niveaux de maturité sont ciblés).

A dbarzin :
On a des experts qualité chez nous, je pourrai demander.

Il faudrait connaître la méthodologie ANSSI sur les processus métiers. Ils ont du bien réfléchir leur modélisation.

Ce n'est pas courant de voir "Opération", s'il ont mis ça c'est surement pour gérer des processus complexes.
La notion d'Activité est fréquente sur un processus.

**Reply by dbarzin** [reply]
@jfcjm  oui, merci.

**Reply by jfcjm** [reply]
Plutôt qu'"enlever", "rendre optionnel" serait possible ?

Rq: Au cas où, j'ai demandé (adresse de contact com') à l'ANSSI s'il leur serait possible de désarchiver le contenu de l'appel à commentaires ... les raisons de la modélisation y sont peut-être mentionnés ...

**Comment by dbarzin** [comment]
Un lien antre processus et opérations a été ajouté et les activités ne sont visible par les utilisateurs qu'au niveau de granularité 3.
